docs/events_handling.md

```diff
@@ -0,0 +1,89 @@
+
+# Events handling
+
+## Subscribed events
+
+Precaution needs to subscribe to a number of events when it's
+to satisfy different use cases of our users.
+
+The events can be separated into two groups `pull requests
+events` and `check suites/runs events`.
+
+### Pull requests events
+
+Official GitHub documentation: [PullRequest event](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
+
+* pull_request.opened - the first event which occurs when a new pull request is created. It's a self-explanatory
```

Comment:
"It's a self-explanatory" - bad grammar.  Don't think this statement adds any value since you already explained it.